tools/docs/test/index.test.mjs

```diff
@@ -0,0 +1,78 @@
+import test from 'node:test';
```

Comment:
We also have a `test/documentation.spec.ts` file, that `@zharinov` is working on here:

- #28590

I don't know if we should split/merge content from either file?

Reply:
I considered to move it, but discarded for now because it's out of scope.

Aldo that test is working on source files where I need to have the docs built before testing